engine/apps/alerts/incident_appearance/renderers/sms_renderer.py

```diff
@@ -24,3 +30,53 @@ def render(self):
             f"integration: {self.alert_group.channel.short_name}, "
             f"alerts registered: {self.alert_group.alerts.count()}."
         )
+
+
+class AlertGroupSMSBundleRenderer(AlertGroupBundleBaseRenderer):
```

Comment:
While we are here do we want to do some further segment optimization on both bundled and non bundled too? Happy to open an issue and do it as a separate task too.  Since our integration names often contain emojis we are getting around 1/2 the characters per segment [(doc)](https://www.twilio.com/docs/glossary/what-sms-character-limit), [Calculator](https://twiliodeved.github.io/message-segment-calculator/).  Potentially we could be much more aggressive with substrings here to ensure we limit segments.  For example `Grafana OnCall: # alert groups @TestOrganization / Grafana Alerting 🦗`

Reply:
That means, that we pay x2/x3 for SMS with emoji, right? 😨
I think we can exclude emojis from SMS. I'm assuming that integrations created after we disabled auto-naming rarely have emoji in the title
Anyway, I'd propose to open a separate issue for that and continue discussion there